### DISCUSSION TITLE
Can't fetch instances nor view videos

### DISCUSSION THREAD

**Discussion by cooolgamer** [libre-tube/LibreTube#6081] [Q&A]
### Steps to reproduce

Open the app, try to play a video from the feed or search or anything.

Go to the settings, instances or launch the app for the first time.

### Expected behavior

The video should play normally and the instances should be fetched correctly

### Actual behavior

The video stay black with no sound and no duration.

I also get "Could not fetch available instances" message when going to instance in settings or launch the app for the first time. The list of prebuilt ones shows up with no uptime and flags

### LibreTube version

v0.23.2

### Android version

Android 6

### Other details

It may be due to the old android version, I have no choice but to stay on it because there's no custom rom at all...
The phone is also rooted with kitsune mask (magisk fork) and have adaway installed

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Runite-Rain** [comment]
Also encountering the same issue. Android 14 on Pixel 8 Pro. Even exempted the app from Adguard to see if this would fix the issue, but still encountering failure to play.

**Comment by Bnyro** [comment]
@Runite-Rain https://github.com/libre-tube/LibreTube/issues/5855, settings -> audio and video -> enable use hls

**Comment by cooolgamer** [comment]
FOUND THE ISSUE:
It was the system certificates, as my phone was on Android 6 and rooted, I copied the ones from my updated pixel 6a over it and it worked.

**Comment by radhealaniesoftware** [comment]
I'm also affecting this since may 12. and since 22 may 2024 could be resolved by selecting/change the instances to coldforge dot xyz. and now it's not working again.
my phone is galaxy a7 2016 with android 7 and its non rooted.

idk what i should i do? tysm.

Edit: its back again after i install the updated system certificate and its back to normal.
